src/OrderFormAcceptanceTests.TestData/SupplierInfo.cs

```diff
@@ -70,5 +70,12 @@ private static IEnumerable<SupplierDetails> SupplierLookup(string connectionStri
 
             return SqlExecutor.Execute<SupplierDetails>(connectionString, query, new { catalogueItemType = (int)catalogueItemType, provisioningType = (int)provisioningType });
         }
+
+        public static int SupplierWithMoreThanOneSolution(string connectionString)
+        {
+            const string query = @"SELECT SupplierId, COUNT(*) FROM CatalogueItem WHERE CatalogueItemTypeId = 1 GROUP BY SupplierId ORDER BY 2 DESC";
```

Comment:
Didn't Sam create some enums somewhere which are more descriptive for the CatalogueItemTypeId

Reply:
Yes, just found them! Will change it to include that. Thanks 😄